app/models/labware_creators/multi_stamp_tubes.rb

```diff
@@ -36,17 +36,24 @@ def allow_tube_duplicates?
     private
 
     def create_labware!
```

Comment:
There's something quite mixed up about this method at the moment. It calls both `create_submission_from_parent_tubes` at the start, and `create_submission_from_child_plate` at the end.
`create_submission_from_parent_tubes` doesn't exist.

Reply:
Ah, it was messed up by this merge: https://github.com/sanger/limber/commit/4d76d3978a667160cdd87e58391c47a087f5cb2c

Reply:
Fixed: https://github.com/sanger/limber/pull/921/commits/f9bf6aa19696f9b9fa523f014327c6636100953d